pkg/bincapz/bincapz.go

```diff
@@ -21,20 +21,23 @@ type Behavior struct {
 
 	DiffAdded   bool `json:",omitempty" yaml:",omitempty"`
 	DiffRemoved bool `json:",omitempty" yaml:",omitempty"`
+
+	// Key is the original map key from map[string]*Behavior
```

Comment:
Update this description as well, to something like:

// ID is a unique behavior identifier